src/State/Users/Db.ts

```diff
@@ -0,0 +1,147 @@
+import { HexKey } from "Nostr";
+import { db as idb, NAME, VERSION } from "Db";
+
+import { UsersDb, MetadataCache, setUsers } from "State/Users";
+import store from "State/Store";
+
+class IndexedDb implements UsersDb {
+  isAvailable() {
+    if ("indexedDB" in window) {
+      return new Promise<boolean>((resolve) => {
+        const req = window.indexedDB.open(NAME, VERSION)
```

Comment:
This seems to create an empty database and not store anything inside?

Reply:
I'll go back to using a dummy `test` db with version 1 then, wdyt?

Reply:
Still doesnt seem to actually use IndexdDb, in logs it says its using indexdb but its not storing anything there

Reply:
My bad, https://github.com/v0l/snort/pull/110/commits/59878776d5a377e62937d1120c3b73a011a91c95 should fix it